pkg/opensearch/response_parser.go

```diff
@@ -131,6 +133,60 @@ func processRawDataResponse(res *es.SearchResponse, timeField string, queryRes b
 	return queryRes
 }
 
+func processRawDocumentResponse(res *es.SearchResponse, timeField, refID string, queryRes backend.DataResponse) backend.DataResponse {
+	documents := make([]map[string]interface{}, len(res.Hits.Hits))
+	for hitIdx, hit := range res.Hits.Hits {
+		doc := map[string]interface{}{
+			"_id":    hit["_id"],
+			"_type":  hit["_type"],
+			"_index": hit["_index"],
```

Comment:
Do  need the `sort` and `highlight` fields that Elasticsearch is copying over?

Reply:
no, these are not in OpenSearch